src/components/CurrencySelector.tsx

```diff
@@ -0,0 +1,116 @@
+import { Payment } from './PaymentCard';
```

Comment:
Do you think we can reuse the `currency selector` for the current story in the sprint in relation to viewing the offers in a users prefereed currency?

Reply:
If so, do you think we can reuse logic/ui?

Reply:
this functionality can be used only in the specific flow - payment with USD as an alternative currency.